src/main/kotlin/nl/info/zac/app/zaak/ZaakRestService.kt

```diff
@@ -1118,6 +1147,29 @@ class ZaakRestService @Inject constructor(
         eventingService.send(ScreenEventType.ZAAK.updated(hoofdZaak.uuid))
     }
 
+    private fun ontkoppelOverigeZaken(
+        zaak: Zaak,
+        andereZaak: Zaak,
+        aardRelatie: AardRelatieEnum,
+        explanation: String
+    ) {
+        zrcClientService.patchZaak(
+            zaakUUID = zaak.uuid,
+            zaak = NillableRelevanteZakenZaakPatch(
+                relevanteAndereZaken = removeRelevanteZaak(zaak.relevanteAndereZaken, andereZaak.url, aardRelatie)
+            ),
+            explanation = explanation
+        )
+//        // Unlink the zaak from the other zaak as well
+//        zrcClientService.patchZaak(
+//            zaakUUID = andereZaak.uuid,
+//            zaak = NillableRelevanteZakenZaakPatch(
+//                relevanteAndereZaken = removeRelevanteZaak(andereZaak.relevanteAndereZaken, zaak.url, aardRelatie)
+//            ),
+//            explanation = explanation
+//        )
```

Comment:
We should ask OpenZaak for comment on this - is this bug or a feature?
![image](https://github.com/user-attachments/assets/64c5cf79-485f-4a06-8335-4aeab4c9300b)